content/en/tracing/other_telemetry/connect_logs_and_traces/php.md

```diff
@@ -24,8 +24,95 @@ further_reading:
 
 ## Automatic injection
 
-Given the many different ways to implement logging in PHP<span class="x x-first x-last">,</span> with some completely circumventing PHP's built-in error-logging API, the Datadog PHP tracing library cannot reliably inject trace and span <span class="x x-first x-last">IDs</span> into logs automatically.
-See the section below to learn how to connect your PHP Logs and traces manually.
+Starting in version `0.89.0`, the PHP tracer automatically injects trace correlation identifiers into application logs. To enable automatic injection, set the environment `DD_TRACE_LOGS_ENABLED` variable (INI setting `datadog.trace.logs_enabled`) to `true`.
+
+The PHP tracer supports PSR-3 compliant loggers, such as [Monolog][4] or [Laminas Log][5].
+
+<div class="alert alert-warning">
+  <strong>Note</strong>: Set up your logging library to produce logs in JSON format so that:
+  <ul>
+    <li>You don't need <a href="/logs/log_configuration/parsing">custom parsing rules</a>.</li>
+    <li>Stack traces are properly wrapped into the log event.</li>
+  </ul>
+</div>
+
+### Configure injection in logs
+
+If you haven't done so already, configure the PHP tracer with `DD_ENV`, `DD_SERVICE`, and `DD_VERSION`. This provides the best experience for adding `env`, `service`, and `version` to your logs (see [Unified Service Tagging][6] for more details).
+
+The PHP tracer provides various ways to configure the injection of trace correlation identifiers into your logs:
+- [Use placeholders in your message](#use-placeholders-in-your-message)
+- [Append the trace correlation identifiers to your message](#append-the-trace-correlation-identifiers-to-your-message)
```

Comment:
I thought we wouldn't document this one for now?

Reply:
Removed this section.